include/spirv-tools/optimizer.hpp

```diff
@@ -838,6 +838,14 @@ Optimizer::PassToken CreateWrapOpKillPass();
 // capabilities.
 Optimizer::PassToken CreateAmdExtToKhrPass();
 
+// Replaces the internal version of GLSLstd450 OpInterpolateAt* instructions
+// with the externally valid version. The internal version allows an OpLoad
+// of the interpolant for the first argument. This pass removes the OpLoad and
+// replace with itspointer. glslang creates the internal version for HLSL. This
+// pass will be called at the end of glslang HLSL legalization after
```

Comment:
Say this will be part of the HLSL legalization.  I would like to see this added to the set of legalization passes, if it is has not already been added.  We should not say that it is at the end because if we may choose to change the order at some point.

Reply:
OK.